CMakeLists.txt

```diff
@@ -45,6 +45,10 @@ if (FIBER_ENABLE_FFTE)
 add_definitions(-DFIBER_ENABLE_FFTE)
 endif()
 
+if (FIBER_ENABLE_FFTW)
+add_definitions(-DFIBER_ENABLE_FFTW)
+endif()
+
```

Comment:
Check commit I made for CMakeLists:
4b2fa7027144bd4029b816cfaa37f9342a9cbff3
This was already like that. 
However, in update from Gerald, edf6510dc4ba811414f36d728f8ed7670c2856a8
It was suggested FFTW read from CMAKE command line, and there was no objection on it. 
Please check it out.

Reply:
The test for FFTW was missing from these commits. Just search for FFTW.
Without these three lines, I always have to add FFTW.
When I add these three lines, FFTW is disabled as it should be.
There is no test in the repo: https://github.com/icl-utk-edu/fiber/blob/master/CMakeLists.txt

Reply:
The version before my PR had just this:
```include_directories(/sw/spack/2021-02-12/opt/spack/linux-scientific7-x86_64/gcc-7.3.0/fftw-3.3.9-xvbumufiowb3nqnifdffjdqga4gqsz2l/include)```
with no conditional and no passing of the precompiler directive, so this is an improvement.

Reply:
Lines copied from the previous PR:
https://github.com/icl-utk-edu/fiber/commit/4b2fa7027144bd4029b816cfaa37f9342a9cbff3

Reply:
Alan, I think you are mistaken in thinking that your previous PR included `add_definitions(-DFIBER_ENABLE_FFTW)`, and my PR did not delete it.  It was never there. 

I only evaluate each PR on its merits relative to the current state of the code base.  If you want to discuss your recent contributions, maybe you should have created a PR instead of directly commiting to master?